ci-operator/step-registry/openshift-qe/installer/bm/day2/label/openshift-qe-installer-bm-day2-label-commands.sh

```
@@ -123,3 +123,12 @@ if [ ${TELCO} == "true" ]; then
     oc get nodes -l node-role.kubernetes.io/worker --show-labels
   fi
 fi
+
+if [ ${NODE_DENSITY_CNI_WORKLOAD}] == "true" ]; then
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this if expression.

(SC1009)